lms/djangoapps/grades/tests/test_grades.py

```diff
@@ -375,14 +464,18 @@ def setUp(self):
         self.request = get_request_for_user(UserFactory())
         self.client.login(username=self.request.user.username, password="test")
         CourseEnrollment.enroll(self.request.user, self.course.id)
+        # The call to get_module_score needs to be warmed up to make the following tests
```

Comment:
This comment is a little bit vague. What is the extra query being made? I'm assuming its result is cached for all the subsequent calls?